process/elasticproc/transactions/transactionDBBuilder.go

```diff
@@ -110,19 +112,76 @@ func (dtb *dbTransactionBuilder) prepareTransaction(
 		ReceiverUserName:  []byte(receiverUserName),
 		SenderUserName:    []byte(senderUserName),
 		IsScCall:          isScCall,
-		Operation:         res.Operation,
```

Comment:
i think we can keep setting `Operation` filed here, because we set it the same on both paths

Reply:
done